### ISSUE TITLE
Include a default imagery layer for Cesium

### ISSUE THREAD

**Issue by robyngit** [NCEAS/metacatui#1938]
In order to [use Cesium maps where we use Google maps](https://github.com/NCEAS/metacatui/issues/1720), we should include a default imagery layer in Cesium maps that doesn't require Cesium Ion or another service. We could 'ship' this default layer with MetacatUI.

**Comment by robyngit** [comment]
If no layers are configured for a new Cesium map, then the Natural Earth II base layer will now be displayed (in branch `feature-#1720-cesium-data-catalog`):

<img width="711" alt="Screen Shot 2022-01-07 at 16 27 05" src="https://user-images.githubusercontent.com/26600641/148610362-7b1dbe47-8677-42db-8c31-e1b0b5d48864.png">

The Natural Earth II layer can now also be added to any map by adding a layer with the following config to the `layers` attribute of a cesium map config: `{  type: 'NaturalEarthII' }`

The images for this layer are shipped with Cesium and thus also shipped with MetacatUI

**Comment by robyngit** [comment]
We never merged the `feature-#1720-cesium-data-catalog` branch into develop or main because the Natural Earth Imagery is quite low resolution and not an ideal default layer. However, I think that this still makes for a reasonable fallback when there are no other options (i.e. it's better than rendering a big blue sphere with no imagery), so I would like to include it as an option.

@jeanetteclark recently discovered [open imagery that is hosted by USGS](https://basemap.nationalmap.gov/arcgis/rest/services/USGSImageryTopo/MapServer). I'm re-opening this issue to cover adding a USGS layer as another default option.

**Comment by robyngit** [comment]
The USGS layer can now be configured by setting a layer asset to:
```
layers: [
  {
    type: 'USGSImageryTopo'
  },
  ...
]
```
The cesium imagery model takes care of setting the other required options (url, attribution, etc).

This is similar to how the Natural Earth layer can be configured:
```
layers: [
  {
    type: 'NaturalEarthII'
  },
  ...
]
```

Here is how the USGS layer looks from a global view:

<img width="710" alt="Screen Shot 2022-08-04 at 11 16 37" src="https://user-images.githubusercontent.com/26600641/182883862-84e3fca2-a19f-4bbe-b5f7-6a253d06fd6e.png">

It's higher resolution than the Natural Earth layer. Here's a random area in Alaska at around 100m scale:

<img width="888" alt="Screen Shot 2022-08-04 at 11 19 19" src="https://user-images.githubusercontent.com/26600641/182884512-0387a95d-6058-40e1-a551-26d5ed64b1fe.png">

The downsides are that the imagery disappears at around the 50-100m scale mark, and that there is no imagery covering the poles:

<img width="710" alt="Screen Shot 2022-08-04 at 11 18 11" src="https://user-images.githubusercontent.com/26600641/182884502-7c28d1f0-a4c5-46cc-a607-f8ef221e07d1.png">
